docs/Developers/Linter_Bears_Advanced.rst

```diff
@@ -41,6 +41,31 @@ The path of the temporary configuration file can be accessed inside
         def create_arguments(filename, file, config_file):
             return "--use-config", config_file
 
+The ``generate_config()`` should not return ``config_file`` if the user wants
```

Comment:
Maybe you should move the part

> If you return ``None``, no configuration file is generated.

here, and then start with "A common case where to explicitly return `None` is ..."

Reply:
Got it 😄